charts/k8s-monitoring/values.yaml

```
@@ -1213,6 +1297,35 @@ grafana-agent-logs:
   # Skip installation of the Grafana Agent CRDs, since we don't use them in this chart
   crds: {create: false}
 
+
+# Settings for the Grafana Agent deployment that gathers profiles.
+# You can use this sections to make modifications to the Grafana Agent deployment.
+# See https://github.com/grafana/agent/tree/main/operations/helm/charts/grafana-agent for available values.
+# @ignored -- This skips including these values in README.md
+grafana-agent-profiles:
+  agent:
+    # This chart is creating the configuration, so the grafana-agent chart does
+    # not need to.
+    configMap: {create: false}
+    clustering: {enabled: true}
+
+    securityContext:
+      privileged: true
+      runAsGroup: 0
+      runAsUser: 0
+
+  controller:
+    type: daemonset
```

This is important for eBPF to work

```suggestion
    type: daemonset
    hostPID: true
```